plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -49,6 +49,61 @@ type Props = {
   fields?: FormProps<any>['fields'];
 };
 
+export function getUiSchemasFromSteps(steps: Step[]): UiSchema[] {
+  const uiSchemas: Array<UiSchema> = [];
+  steps.forEach(step => {
+    if (!step.schema || !step.schema.properties) return;
```

Comment:
```suggestion
    if (!step.schema || !step.schema.properties) return [];
```

Should this return an array? I am concerned that the code will throw `find is not a function` few lines below, when trying to call `uiSchemas.find`.

Reply:
Makes sense!!! Committing  your suggestion...